tst/Pester.RSpec.Configuration.ts.ps1

```diff
@@ -1116,6 +1120,23 @@ i -PassThru:$PassThru {
             $r = Invoke-Pester -Configuration $c
             $r.Configuration.Output.CIFormat.Value | Verify-Equal "None"
         }
+
+        t "Output.CILogLevel is Error when set" {
```

Comment:
Please move this to it's own `b 'Output.CILogLevel'`-block (think of `b` as `Context`) and include tests for each option each setting + exception when invalid value.

See `b "Run.SkipRemainingOnFailure"` as example.

Reply:
Not sure why the pipeline keeps failing. Added tests based on `b "Run.SkipRemainingOnFailure"`.

From the pipeline logs:

```
2023-02-14T22:31:11.2213015Z [91m  - a[0m
2023-02-14T22:31:11.2291056Z [n] - Exception is thrown when incorrect option is set -> An exception was expected, but no exception was thrown! 
2023-02-14T22:31:11.2293542Z   at Verify-Throw, /home/vsts/work/1/s/tst/axiom/Verify-Throw.ps1:17 
2023-02-14T22:31:11.2296024Z   at <ScriptBlock>, /home/vsts/work/1/s/tst/Pester.RSpec.Configuration.ts.ps1:1159 
```

Reply:
Added 👍🏻